test/e2e.spec.js

```diff
@@ -42,7 +42,7 @@ ruleTester.run('compat', rule, {
     },
     {
       code: 'new IntersectionObserver(() => {}, {});',
-      settings: { browsers: ['chrome 57'] }
+      settings: { browsers: ['chrome 58'] }
```

Comment:
Don't know why this didn't fail before, but according to caniuse chrome 58 is the first version that completely supports intersectionobserver